book/accessibility.md

```diff
@@ -587,24 +593,21 @@ white text on a black background.
 
 ::: {.further}
 
-Dark mode is a relatively recent browser feature. In the original design of CSS,
-the [cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade) defined
-not just browser and author style sheets, but also [*user*][user-style] style
-sheets. These are style sheets defined by the person using the browser, as a
-kind of custom theme. Another approach is to add a
-[browser extension][extension] (or equivalent browser built-in feature) that
-injects additional style sheets applying dark styles.[^no-user-styles]
-
-With one of these mechanisms, users might be able to add their
-own dark mode. While it's relatively easy for this to work well overriding the
-browser's default style sheet and a few common sites, it's very hard to come up
-with styles that work well alongside the style sheets of many sites without
-losing readability or failing to provide adequate dark mode styling.
-
-[extension]: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions
-[user-style]: https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#user_stylesheets
-[^no-user-styles]: Most browsers these days don't even support user style
-sheets, and instead rely on extensions.
+Of course, a real browser needs change quite a bit more colors than
+our browser---scroll bars, input elements, menus, and so on.
+The browser should really not be changing colors on unsuspecting
+pages. For example, it'll likely have terrible accessibility outcomes!
+Instead web pages [indicate support][dark-mode-post] for dark mode
+using the `color-scheme` [`meta` tag][meta-tag] or [CSS
+property][css-prop]. Before `color-scheme` was standardized, web pages
+could offer alternative color schemes using [alternative style
+sheets][alt-style], but few browsers supported it (of the major ones,
```

Comment:
Was this feature in firefox available for dark mode? I didn't know that.

Reply:
Uhh—the website could offer a separate darker style sheet if it wanted to, and I know a few websites offered alternative high-contrast stylesheet, but it didn't directly have anything to do with dark mode or high contrast or whatever.

Reply:
Great suggestion.